docs/debugging.md

```diff
@@ -79,3 +79,53 @@ Open a gdb session -> start -> nop
 Done!
 
 ![pycharm9](https://github.com/hugsy/gef/assets/9882181/b22ec431-57e7-442a-835e-5817bdac7687)
+
+
+## Debugging with VSCode
```

Comment:
IMO this chapter should be first, VSCode its more famous/used than Pycharm, don you?

![image](https://github.com/hugsy/gef/assets/9882181/4f6623cb-d6f7-4b98-870e-48170d72d15b)

Reply:
Doesn't matter to me, as long as it is all searchable (and it is). And PyCharm comes before VsCode alphabetically:)